code/languages/C++/flow-control/README.md

```diff
@@ -0,0 +1,203 @@
+# Flow Control
+A simple C++ statement is each of the individual instructions of a program, like the variable declarations and expressions seen in previous sections. They always end with a semicolon (;), and are executed in the same order in which they appear in a program. But programs are not limited to a linear sequence of statements. During its process, a program may repeat segments of code, or take decisions and bifurcate. For that purpose, C++ provides flow control statements that serve to specify what has to be done by our program, when, and under which circumstances.
+
+Program statements that cause such jumps are called control statements. There are two major categories:
+
+* Loops
+* Decisions
+
+## Loops
+Loops repeat a statement a certain number of times, or while a condition is fulfilled. They are introduced by the keywords while, do, and for.
+
+### The while loop
+Its syntax is:
+
+``` 
+while (expression) statement
+```
+
+The while-loop simply repeats statement while expression is true. If, after any execution of statement, expression is no longer true, the loop ends, and the program continues right after the loop. For example, let's have a look at a countdown using a while-loop:
+``` C++
+// custom countdown using while
+#include <iostream>
+using namespace std;
```

Comment:
Could you remove this?

Reply:
Sir, exactly which all lines?

Reply:
I have made the changes. Kindly review it.